recipes/pyuvdata/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyuvdata" %}
+{% set version = "1.2.1" %}
+{% set sha256 = "8ac8cc6467ff1d73814418bd699dad28fd9f278908f75ea038f20f7821e1b1b5" %}
+
+package:
+  name: {{ name }}
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  #git_url: https://github.com/HERA-Team/pyuvdata.git
+  #git_rev: conda_test
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=files.txt
+  skip: true  # [win or py3k]
+
+requirements:
+  build:
+    - aipy >=2.1.6
+    - astropy >=1.2
+    - numpy >=1.11
+    - pyephem
+    - python <3
+    - scipy
+    - setuptools
+  run:
+    - aipy >=2.1.6
+    - astropy >=1.2
+    - numpy >=1.11
+    - pyephem
+    - python <3
+    - scipy
+    - setuptools
```

Comment:
Is `setuptools` an actual runtime req? I didn't see `pkg_resources` used anywhere in the source code.